### ISSUE TITLE
mdSelect: accessibility label lost when item is selected

### ISSUE THREAD

**Issue by marcysutton** [angular/material#1893]
Because of the way `mdSelect` dynamically binds `aria-labelledby` to the `<md-select-label>`, when an option is selected by the user or programmatically by default, there is no descriptive label telling a screen reader user what the select is for. See the "select label test" portion of this demo: http://codepen.io/marcysutton/full/ZYqjPj/

In VoiceOver, the native select is announced as "Animal Muppets pop-up button", combining the first option and the label of the component. The mdSelect component should announce something similar.
